src/libs/ReportUtils.ts

```diff
@@ -1473,7 +1473,14 @@ function canShowReportRecipientLocalTime(personalDetails: OnyxCollection<Persona
     const reportRecipient = personalDetails?.[reportRecipientAccountIDs[0]];
     const reportRecipientTimezone = reportRecipient?.timezone ?? CONST.DEFAULT_TIME_ZONE;
     const isReportParticipantValidated = reportRecipient?.validated ?? false;
-    return Boolean(!hasMultipleParticipants && !isChatRoom(report) && !isPolicyExpenseChat(report) && reportRecipient && reportRecipientTimezone?.selected && isReportParticipantValidated);
+    return Boolean(
+        !hasMultipleParticipants &&
+            !isChatRoom(report) &&
+            !isPolicyExpenseChat(getRootParentReport(report)) &&
```

Comment:
Does this change prevent the local time of the approver from being displayed even when you're exchanging messages with them inside an expense report?

Reply:
Inside policy expense chat, because it is shared with workspace owner + admins.
If a workspace contains two users - owner + admin, then currently we are not displaying another user's local in the main policy expense chat view, but do display time in the expense view and transaction thread. So this fixes that dissimilarity.
Given there can be multiple approvers, we don't know which user's time to display.

Reply:
I didn't know that all admins have access to the expense chat. I was assuming that there are only two participants in an expense chat.

Thank you for the explanation 🙇 \